test/e2e/run_env_test.go

```diff
@@ -58,6 +58,13 @@ var _ = Describe("Podman run", func() {
 		Expect(session).Should(Exit(0))
 		Expect(session.OutputToString()).To(ContainSubstring("/bin"))
 
+		// Verify environ keys with spaces do not blow up podman command
+		os.Setenv("FOO BAR", "BAZ")
+		session = podmanTest.Podman([]string{"run", "--rm", ALPINE, "true"})
```

Comment:
Why not go for a full-on all-the-way test?
```go
    ...{"run", "--rm", "-e", "FOO BAR", ALPINE, "printenv", "FOO BAR"})
   ...output-should-equal("BAZ")
```
(for extra credit, `s/BAZ/BAZ OOKA or something with spaces/`)

Reply:
I could do that but that would be a separate test.  What I was looking at was a failure to run if any environment variable contained a space, even if the variable was not being passed to the container.

Reply:
I also did not see an easy way to do this in a system test.

env "foo bar"="foo baz" run_podman ...

Blew up.

Reply:
I don't see a sane way to test this in system tests either. Bash won't allow it. It would be possible to build a test binary that uses `prctl()` to inject `FOO BAR` into the parent process's `environ`, but let's just not go there.